articles/install-guide/index.md

```diff
@@ -9,13 +9,44 @@ uid: microsoft.quantum.install
 
-* **[Q# Notebooks](https://github.com/Microsoft/Quantum/tree/master/Samples/src/IntroToIQSharp/Notebook.ipynb)**: explains how to compile and simulate Q# operations inside a Jupyter notebook.
-* **[Teleport](https://github.com/Microsoft/Quantum/tree/master/Samples/src/Teleportation/Notebook.ipynb)**: shows how to implement the teleport algorithm using Q#.
+### Jupyter Notebooks ###
 
-To run these interactively, install IQ# on your machine (see below) and take a look at our [Jupyter notebooks getting started guide](xref:microsoft.quantum.install.jupyter).
+ A great way to get started with Q# before installing the Quantum Development Kit is via some of the Jupyter notebooks available in our [GitHub Quantum repository](https://github.com/Microsoft/Quantum.git).  In particular:
 
+* **[Q# Notebooks](https://github.com/Microsoft/Quantum/tree/master/Samples/src/IntroToIQSharp/Notebook.ipynb)** [(Run online)](https://mybinder.org/v2/gh/Microsoft/Quantum/master?filepath=Samples%2Fsrc%2FIntroToIQSharp%2FNotebook.ipynb) : explains how to compile and simulate Q# operations inside a Jupyter notebook.
+* **[Teleport](https://github.com/Microsoft/Quantum/tree/master/Samples/src/Teleportation/Notebook.ipynb)** [(Run online)](https://mybinder.org/v2/gh/microsoft/Quantum/master?filepath=Samples%2Fsrc%2FTeleportation%2FNotebook.ipynb): shows how to implement the teleport algorithm using Q#.
+
+To run these locally, install IQ# on your machine (see below) and take a look at our [Jupyter notebooks getting started guide](xref:microsoft.quantum.install.jupyter).
+
+### Visual Studio ###
+
+ If you are already a [Visual Studio](https://visualstudio.microsoft.com/vs/) user, you can install the extension to get started developing Q# in Visual Studio. 
+> [!div class="button"]
+> [Visual Studio extension](https://marketplace.visualstudio.com/items?itemName=quantum.DevKit)
+
+Then, check our [Quickstart guide for step-by-step](quickstart?tabs=tabid-vs2017) instructions for your first quantum program.
+
+### Visual Studio Code ###
+
+ If you are already a [Visual Studio Code](https://code.visualstudio.com/) user, you can install the extension to get started developing Q# in Visual Studio Code.
+> [!div class="button"]
+> [Visual Studio Code extension](https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode)
+
+Then, check our [Quickstart guide for step-by-step](quickstart?tabs=tabid-vscode) instructions for your first quantum program.
+
+
+### Python ###
+
+If you are a Python user, check out our [Getting Started with Python and the Quantum Development Kit guide](xref:microsoft.quantum.install.python).
+
+## Learning Quantum Computing with Q# ##
+
+* [Quantum Katas](https://github.com/Microsoft/QuantumKatas)
+* [Samples](https://github.com/Microsoft/Quantum)
+* [Quantum Computing Concepts](xref:microsoft.quantum.concepts.intro)
+* [Microsoft Quantum Blog](https://cloudblogs.microsoft.com/quantum/?ext)
```

Comment:
Should we link to the devblog here as well? https://devblogs.microsoft.com/qsharp/

Reply:
Done

Reply:
oh yeah, I like this as a resource, but I'm not sure what to call it to differentiate from the other quantum blog mentioned. Is Microsoft Quantum Developer Blog or Q# blog sufficient for this?